script.js

```diff
@@ -1,36 +1,32 @@
 /*===== TOGGLE ICON NAVBAR =====*/
-let menuIcon = document.querySelector('#menu-icon');
```

Comment:
Uma boa prática, é escolher const ao invés de let se sua variável nunca for reatribuida, ou seja, nunca for alterada. Escolha sempre const nesses casos, como aparenta ser no seu projeto.